cli/alterations/alteration.py

```diff
@@ -17,52 +17,54 @@
 
 from apis import nifiManagment
 from apis import auxiliaryFunctions
-
+import env
 
 def createMerge(nifiConfiguration,information, name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Merge.json",information)
-    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information["maxMessages"])
+    merge = auxiliaryFunctions.addSensitiveVariable(merge, "MergeContent", "Maximum Number of Entries", information[env.ALTERATION_MAX_MESSAGES_TAG])
     nifiConfiguration.create(nameCompose, merge)
-    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information["windowSeconds"])
+    nifiConfiguration.changeSchedule(nameCompose, "MergeContent", information[env.ALTERATION_WINDOW_SECONDS_TAG])
 
 
 def createDecode(nifiConfiguration,information,name):
-    nameCompose= nameActionReturn(information["action"],name)
+    nameCompose= nameActionReturn(information[env.ALTERATION_ACTION_TAG],name)
     merge=auxiliaryFunctions.prepareforAll("./template/alterations/Encode_Decode.json",information)
     merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Mode", "Decode")
-    if "Encoding" in information:
-        merge = auxiliaryFunctions.addSensitiveVariable(merge, "EncodeContent", "Encoding", information["Encoding"])
+    if env.ALTERATION_ENCODING_TAG in information:
```

Comment:
If `env.ALTERATION_ENCODING_TAG`is not in `information`, `nameCompose` would retrieve a wrong value. Please, ask yourself what happens if `information` doesn't contain that key.

Reply:
If `env.ALTERATION_ENCODING_TAG` is not in `information`. The alteration workflow won't be deployed. Because this tag is checked before.
https://github.com/interTwin-eu/dcnios/blob/dev-slangarita/cli/alterations/alteration.py#L51